packages/lockfile-lint/README.md

```diff
@@ -59,7 +59,7 @@ lockfile-lint --path yarn.lock --allowed-hosts registry.yarnpkg.com --validate-h
 **Example 3**: allow the lockfile to contain packages served over github and so need to specify github.com as a host as well as the `git+https:` as a valid URI scheme
 
 ```bash
-lockfile-lint --path yarn.lock --allowed-hosts yarn github.com --validate-https --allowed-schemes "https:" "git+https:"
+lockfile-lint --path yarn.lock --allowed-hosts yarn github.com --allowed-schemes "https:" "git+https:"
```

Comment:
I've merged this change already with @juanpicado on #90 but there are other issues that need fixing on the README in this section so I'll lay them out so you can fix too :)

Reply:
can you update line 66 for the `allowed-schemes` bullet to read this:
```
- `--allowed-schemes` is used instead of `validate-https` and it explicitly allows both `https:` and `git+https:` as the [HTTP Scheme](https://tools.ietf.org/html/rfc3986#section-3.1) for the github URL. Note that `--allowed-schemes` and `--validate-https` are mutually exclusive.
```